en/08_Changelogs/6.2.0.md

```diff
@@ -8,11 +8,18 @@ title: 6.2.0 (unreleased)
 
 - [Features and enhancements](#features-and-enhancements)
   - [Other new features and enhancements](#other-new)
+  - [Accessibilty improvements](#accessibilty-improvements)
 - [API changes](#api-changes)
 - [Bug fixes](#bug-fixes)
 
 ## Features and enhancements
 
+### Accessibilty improvements
+
```

Comment:
Maybe put a blurb here, something like "this release has focused on improving accessibility, some of the accessibility improvements have required changing selectors you may be using in your JavaScript or CSS customisations"